install/kubernetes/helm/istio/charts/gateways/values.yaml

```diff
@@ -113,7 +113,7 @@ istio-ingressgateway:
   nodeSelector: {}
 
 istio-egressgateway:
-  enabled: true
+  enabled: false
```

Comment:
if you turn this off, you have to update all the egress tasks to deploy the egress gateway

Reply:
It is turned back on in the demo profile, which all tasks use.